keel-sql/src/main/resources/db/changelog/20210318-versioned-environments.yml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+- changeSet:
+    id: versioned-envrironments
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: environment
+        columns:
+        - name: version
+          type: integer
+          defaultValueNumeric: 1
+          afterColumn: name
+          constraints:
+            nullable: false
+    - dropForeignKeyConstraint:
+        baseTableName: environment
+        constraintName: fk_environment_delivery_config
+    - dropUniqueConstraint:
+        tableName: environment
+        constraintName: environment_delivery_config_name_idx
```

Comment:
For some reason you have to drop the FK before MySQL will let you drop the unique index 🤷